bedrock/firefox/templates/firefox/whatsnew/index.html

```diff
@@ -21,7 +21,7 @@
 {% block site_header %}{% endblock %}
 
 {% block page_css %}
-  {% stylesheet 'firefox_whatsnew_54' %}
+  {% stylesheet 'firefox_whatsnew' %}
```

Comment:
@flodolo is there anything you'd like to do here to tidy up lang files? The file here contains the following above:

```
{% add_lang_files "firefox/whatsnew_50" "firefox/sendto" %}
```

Reply:
Yes, I was just noticing that. I would rename it to firefox/whatsnew for making it less confusing, then we can get rid of firefox/whatsnew50